### PR TITLE
refactor(utils): Simplify align/justify

### PR DESCRIPTION
**Pull request by obartra**
Simplify `getJustify` and `getAlignment` methods

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -17,24 +17,26 @@ export function times(number = 0, callback = noop) {
...
-      return `${prefix}bottom`
+      return `${prefix}--bottom`
     default:
       return ''
   }
 }
 
-export function getJustify(value, prefix) {
+export function getJustify(value) {
+  const prefix = 'grid'
```
Comment (arahansen) [inline src/utils.js]: might as well move this to the top of the file

Comment (obartra) [inline src/utils.js]: hm this is specific to justify not to utils though

Comment (arahansen) [inline src/utils.js]: But creating new variables  on every call Oscar! Every call!!

Comment (arahansen) [inline src/utils.js]: But no, you're right. I concede 😄

Comment (obartra) [inline src/utils.js]: haha well if we are going to get into optimizations, external variables will de-opt the function from optimization!!

**Pull request merged**